tests/net/traffic_class/testcase.yaml

```diff
@@ -1,177 +1,177 @@
 tests:
   test_tc_1:
-    min_ram: 80
+    platform_whitelist: qemu_x86
```

Comment:
I just checked and this test runs just fine in native_posix, maybe that is the better option here, this way we will get code coverage information.

also, you can add

common:
  platform_whitelist: <platform>
  tags: net traffic_class 

on the top and avoid adding this per testcase.